pyproject.toml

```diff
@@ -96,9 +96,39 @@ bson=["py.typed", "*.pyi"]
 pymongo=["py.typed", "*.pyi"]
 gridfs=["py.typed", "*.pyi"]
 
+[tool.pytest.ini_options]
+minversion = "7"
+addopts = ["-ra", "--strict-config", "--strict-markers", "--junitxml=xunit-results/TEST-results.xml"]
+testpaths = ["test"]
+log_cli_level = "INFO"
+norecursedirs = ["test/*"]
+faulthandler_timeout = 1500
+xfail_strict = true
+filterwarnings = [
+    "error",
+    "module:use an explicit session with no_cursor_timeout=True:UserWarning",
+    "module:serverselectiontimeoutms must be:UserWarning",
+    "module:Unsupported compressor:UserWarning",
+    "module:zlibcompressionlevel must be:UserWarning",
+    "module:unclosed <socket.socket:ResourceWarning",
```

Comment:
Can you add a comment saying to remove this after PYTHON-3923?

Reply:
Done.